openapi/components/schemas/Company.yaml

```
@@ -0,0 +1,50 @@
+type: object
+description: |-
+  Company information that is associated with the customer's primary email address domain.
+  
+  This is a paid feature, to enable it [contact Rebilly](https://www.rebilly.com/support/).
+nullable: true
+readOnly: true
+properties:
+  name:
+    type: string
+    description: Name of the company.
+    nullable: true
+  domain:
+    type: string
```

Issue: We need a description for this field. (I'm not sure if my suggestion is correct)
```suggestion
    type: string
    description: Website domain of the company.
```